cinema/urls.py

```diff
@@ -11,13 +11,18 @@
 )
 
 router = routers.DefaultRouter()
-router.register("genres", GenreViewSet)
-router.register("actors", ActorViewSet)
-router.register("cinema_halls", CinemaHallViewSet)
-router.register("movies", MovieViewSet)
+
 router.register("movie_sessions", MovieSessionViewSet)
-router.register("orders", OrderViewSet)
 
-urlpatterns = [path("", include(router.urls))]
+
+urlpatterns = [
+    path("", include(router.urls)),
+    path("genres/", GenreViewSet.as_view({"get": "list", "post": "create"}), name="genre-list"),
+    path("actors/", ActorViewSet.as_view({"get": "list", "post": "create"}), name="actor-list"),
+    path("cinema_halls/", CinemaHallViewSet.as_view({"get": "list", "post": "create"}), name="cinemahall-list"),
+    path("orders/", OrderViewSet.as_view({"get": "list", "post": "create"}), name="order-list"),
+   path("movies/", MovieViewSet.as_view({"get": "list", "post": "create"}), name="movie-list"),
+   path("movies/<int:pk>/", MovieViewSet.as_view({"get": "retrieve"}), name="movie-detail"),
```

Comment:
These paths manually define the views for 'genres', 'actors', 'cinema_halls', 'orders', and 'movies'. If you intend to use the router for these viewsets, you should register them with the router instead of defining them manually. This would simplify the URL configuration.

Reply:
using for actions available for views